eng/helix/content/runtests.sh

```diff
@@ -1,32 +1,20 @@
 #!/usr/bin/env bash
 
-dotnet_sdk_version="$2"
-dotnet_runtime_version="$3"
-helixQueue="$5"
-installPlaywright="${10}"
+helixQueue="$3"
+installPlaywright="$8"
 
 RESET="\033[0m"
 RED="\033[0;31m"
 YELLOW="\033[0;33m"
 MAGENTA="\033[0;95m"
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-# Ensures every invocation of dotnet apps uses the same dotnet.exe
-# Add $random to path to ensure tests don't expect dotnet to be in a particular path
-export DOTNET_ROOT="$DIR/.dotnet$RANDOM"
-
-# Ensure dotnet comes first on PATH
-export PATH="$DOTNET_ROOT:$PATH:$DIR/node/bin"
-
 # Prevent fallback to global .NET locations. This ensures our tests use the shared frameworks we specify and don't rollforward to something else that might be installed on the machine
 export DOTNET_MULTILEVEL_LOOKUP=0
-
-# Avoid contaminating userprofiles
-# Add $random to path to ensure tests don't expect home to be in a particular path
-export DOTNET_CLI_HOME="$DIR/.home$RANDOM"
```

Comment:
Restore this line.

Reply:
Oops, forgot about the `--tool-path` conversation: I don't think that changes where `dotnet` _looks_ for tools. Need `DOTNET_CLI_HOME` to handle both installation and use.

Reply:
I'm not sure what the net result is here then?  We don't install anything to DOTNET_CLI_HOME so does setting it matter?  We install the tools to the helix workitem root, in the old days we didn't do this so the CLI_HOME might have mattered.

Reply:
Hmm, you are likely correct this is a holdover. Can clean up https://github.com/dotnet/aspnetcore/blob/5a01c6fb9078402dd69c1500f7e1ea7862a20696/eng/helix/content/RunTests/TestRunner.cs#L167-L171. On the other hand, may need `HELIX_WORKITEM_ROOT` in the `Path` if it's not there already to find the newly-installed tools.

One thing to check: I doubt `TestRunner` sets `DotNetEfFullPath` correctly in the environment because
- https://github.com/dotnet/aspnetcore/blob/1d7443d1823a63b60f94ec9e93b1c4ca8e635aae/eng/helix/content/RunTests/TestRunner.cs#L42 won't work on Linux or macOS
- [that line](https://github.com/dotnet/aspnetcore/blob/1d7443d1823a63b60f94ec9e93b1c4ca8e635aae/eng/helix/content/RunTests/TestRunner.cs#L42) shouldn't contain both `helixDir` and `nugetRestore` because both are full paths. `helixDir` alone is probably right

Should add some debug output or use a binary log to see where `dotnet ef` and `dotnet serve` are found.

Reply:
Template tests currently are windows only so there's no expectation that they would work on non windows right now, so that makes sense

Reply:
But yeah we need the workitem root in the path for the tools, I'll get rid of the CLI_HOME I'll leave the comment about the path separator since its quite possible we will need to muck with the path again in the future